utils/luci-app-ffwizard-berlin/root/www/cgi-bin/has_root-pass

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+echo "Content-type: text/plain"
+echo ""
+USER=root
+# taken from http://serverfault.com/a/240961
+grep ${USER} /etc/shadow | awk -F":" '{ if ($2 == "!" || $2 == "*" || $2 == "") \
```

Comment:
Yes please. This is how security issues happen. Also, in this case it's actually a bug: User locks root account, script says "no password set", wizard adds password to URL, webserver throws the well-known exception.